mopro-core/build.rs

```diff
@@ -71,12 +71,12 @@ fn read_config() -> Result<Config> {
             println!("cargo:warning=BUILD_CONFIG_PATH not set. Using default configuration.");
             let default_config = r#"
                 [circuit]
-                dir = "examples/circom/keccak256"
-                name = "keccak256_256_test"
+                dir = "examples/circom/multiplier2"
+                name = "multiplier2"
 
                 [dylib]
                 use_dylib = false
-                name = "keccak256.dylib"
+                name = "multiplier2.dylib"
```

Comment:
it looks like the mopro-core tests takes `multiplier2` for `test_generate_proof2`
so the inputs and outputs don't match keccak256 proof